internal/pkg/build/sources/conveyorPacker_busybox.go: busybox exec format error

## Version of Singularity:

I am getting a lot of errors during CI due to a `exec format error` with `busybox` from `pkg/build/remoteBuilder`. I also see the problem when running `make testall`.

### Expected behavior

They should all pass.

### Actual behavior

```
github.com/sylabs/singularity/internal/pkg/build/remotebuilder	1.019s	coverage: 8.5% of statements
--- FAIL: TestBusyBoxConveyor (1.31s)
    conveyorPacker_busybox_test.go:51: failed to Get from ../../../../examples/busybox/Singularity: While performing busybox install: fork/exec /tmp/sbuild-busybox-840806805/fs/bin/busybox: exec format error
FAIL
coverage: 9.7% of statements
FAIL	github.com/sylabs/singularity/internal/pkg/build/sources	1.394s
```

### Steps to reproduce behavior

`make testall` on Ubuntu Cosmic or CI

--- a/examples/busybox/Singularity
+++ b/examples/busybox/Singularity
@@ -1,5 +1,5 @@
 BootStrap: busybox
-MirrorURL: https://www.busybox.net/downloads/binaries/1.26.1-defconfig-multiarch/busybox-x86_64
+MirrorURL: https://www.busybox.net/downloads/binaries/1.26.2-defconfig-multiarch/busybox-x86_64
 
 %post
     echo "Hello from inside the container"

--- a/examples/instances/Singularity
+++ b/examples/instances/Singularity
@@ -1,5 +1,5 @@
 BootStrap: busybox
-MirrorURL: https://www.busybox.net/downloads/binaries/1.26.1-defconfig-multiarch/busybox-x86_64
+MirrorURL: https://www.busybox.net/downloads/binaries/1.26.2-defconfig-multiarch/busybox-x86_64
 
 %startscript
     exec nc -ll -p $1 -e /bin/cat 2>/dev/null

